sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/transfer/TransferIdInfoLiveData.java

```diff
@@ -35,6 +35,9 @@
     private final MutableLiveData<TransferIdOrError> transferIdOrErrorLiveData = new MutableLiveData<>();
     // the output TransferInfo LiveData.
     private final MediatorLiveData<TransferInfo> transferInfoLiveData = new MediatorLiveData<>();
+    // the the object that TransferClient methods update any state that it want TransferInfo LiveData source
```

Comment:
I don't understand this comment, can you maybe reword? It's not clear to me at this point what the TransferFlags are for. Maybe this encapsulates the different states a transfer can be in?